Number_types/include/CGAL/FPU.h

```diff
@@ -114,9 +114,13 @@ extern "C" {
 
 // Only define CGAL_USE_SSE2 for 64 bits where malloc has a suitable
 // alignment, 32 bits is too dangerous.
-#if defined(CGAL_HAS_SSE2) && (defined(__x86_64__) || defined(_M_X64))
+#if defined CGAL_HAS_SSE2
+#if defined __x86_64__ || defined _M_X64
+#if !defined CGAL_ALWAYS_ROUND_TO_NEAREST
```

Comment:
What's wrong with `&&`? You can still write it on several lines if you find it more readable (ending a line with `\`).

Reply:
Sounds good.